### PR TITLE
Update carbon-componts version

### PR DESCRIPTION
**Pull request by zvonimirfras**

### CODE REVIEW

File: packages/angular/package.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
     "lodash-es": "4.17.15",
     "ng-packagr": "11.1.4",
     "rxjs": "6.5.3",
-    "sass": "^1.32.11",
+    "sass": "1.36.0",
```
Comment (davidicus) [inline packages/angular/package.json:90]: Is there something preventing us from upgrading to the latest version? In the react directory I see we are rolling back the sass version.

File: packages/monorepo-utils/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "fs-extra": "9.1.0",
     "glob": "7.1.6",
     "rimraf": "3.0.2",
-    "sass": "1.32.7"
+    "sass": "1.36.0"
```
Comment (davidicus) [inline packages/monorepo-utils/package.json:14]: same comment about sass versions

File: packages/react/package.json [hunk 3]
```diff
@@ -331,7 +331,7 @@
     "rollup-plugin-filesize": "^9.1.1",
     "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-uglify": "^6.0.4",
-    "sass": "^1.42.1",
+    "sass": "1.36.0",
```
Comment (davidicus) [inline packages/react/package.json:334]: Is there a reason we need to roll back the sass version?

File: packages/styles/package.json [hunk 4]
```diff
@@ -29,11 +29,11 @@
     "@carbon/motion": "10.12.0",
     "@carbon/themes": "10.26.0",
     "ai-apps-monorepo-utils": "^2.146.1-next.0",
-    "carbon-components": "10.20.0",
+    "carbon-components": "10.47.0",
     "react-grid-layout": "1.2.0",
     "react-resizable": "1.11.0",
     "rimraf": "3.0.2",
-    "sass": "1.42.1"
+    "sass": "1.36.0"
```
Comment (davidicus) [inline packages/styles/package.json:36]: Same comment about sass versions

**Review by kevinsperrine — COMMENTED** [review]
I see a lot of css issues in storybook, but I don't have anything to compare it to to know if these are regressions or previous issues? Do you have a domain like https://next.carbon-addons-iot-react.com/ to compare angular against?

Screen Shot 2021-11-15 at 4 40 51 PM

Screen Shot 2021-11-15 at 4 40 29 PM

Screen Shot 2021-11-15 at 4 40 03 PM

File: packages/angular/package.json (hunk 1 above)
Comment (zvonimirfras) [reply packages/angular/package.json:90]: iirc later version failed to build due to dividing with a slash. This version just throws a warning but doesn't fail the build.

Comment (davidicus) [reply packages/angular/package.json:90]: I think that division should be supported until the next major version
```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
```

Comment (davidicus) [reply packages/angular/package.json:90]: We are currently using the version of Sass that you rolled back in the React repo.

**Comment by zvonimirfras** [comment]
> I see a lot of css issues in storybook, but I don't have anything to compare it to to know if these are regressions or previous issues? Do you have a domain like https://next.carbon-addons-iot-react.com/ to compare angular against?
> 
> Screen Shot 2021-11-15 at 4 40 51 PM
> 
> Screen Shot 2021-11-15 at 4 40 29 PM
> 
> Screen Shot 2021-11-15 at 4 40 03 PM

Seems fine 🤷  -> https://deploy-preview-3023--ai-apps-pal-angular.netlify.app/

**Comment by kevinsperrine** [comment]
> > I see a lot of css issues in storybook, but I don't have anything to compare it to to know if these are regressions or previous issues? Do you have a domain like https://next.carbon-addons-iot-react.com/ to compare angular against?
> 
> Seems fine 🤷 -> https://deploy-preview-3023--ai-apps-pal-angular.netlify.app/

This is fine? The menu button doesn't even work and all the items are always visible.
Screen Shot 2021-11-29 at 9 12 57 AM

**Comment by zvonimirfras** [comment]
> > > I see a lot of css issues in storybook, but I don't have anything to compare it to to know if these are regressions or previous issues? Do you have a domain like https://next.carbon-addons-iot-react.com/ to compare angular against?
> > 
> > 
> > Seems fine 🤷 -> https://deploy-preview-3023--ai-apps-pal-angular.netlify.app/
> 
> This is fine? The menu button doesn't even work and all the items are always visible.

🤔 that's not what it looked like for me when I posted. Might be caching.

**Review by kevinsperrine — APPROVED** [review]
Looks good, now!

**Review by davidicus — APPROVED** [review]

**Pull request merged**